PukingPredator/Assets/Scripts/Player.cs

```diff
@@ -38,6 +38,7 @@ private void Awake()
         dash.onComplete += DashFinished;
 
         movement = GetComponent<Movement>();
+        AudioManager.Instance.PlayBackground();
```

Comment:
i dont think it makes a ton of sense to have this here long term, but its fine for now